src/layers/conv.jl

```diff
@@ -187,6 +187,23 @@ with ``\hat{L}`` the [`scaled_laplacian`](@ref).
 - `k`: The order of Chebyshev polynomial.
 - `bias`: Add learnable bias.
 - `init`: Weights' initializer.
+
+# Examples
+
+```julia
+
+# create data
+s = [1,1,2,3]
+t = [2,3,1,1]
+g = GNNGraph(s, t)
+x = randn(3, g.num_nodes)
```

Comment:
```suggestion
x = randn(Float32, 3, g.num_nodes)
```
This should be changed everywhere.

Reply:
Done